### ISSUE TITLE
Minor accessibility issues with general layout

### ISSUE THREAD

**Issue by selfthinker** [splitbrain/dokuwiki#3463]
* [ ] The page ID tab looks highlighted and shows it has a connection to the main content but that is not programmatically determinable.
It could at least have some hidden text, like "Page ID: ", to explain what it is and what the connection is to the page.
That fails [1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships).
![](https://user-images.githubusercontent.com/108893/113616088-9536a780-964c-11eb-989d-52411568b214.png)

## Other non-fail accessibility issues

These don't strictly fail WCAG but are still accessibility issues. Fixing them is recommended but optional for passing WCAG.

* [ ] The header, footer, navigation and main content should be marked up as such (with `header`, `footer`, `nav` and `main`) to help screen reader users navigate the page.
This was fixed in #3432.

* [ ] Repeated blocks (tools, TOC, sidebar) have hidden headings but they are not in hierarchy. This could confuse screen reader users a bit, especially when those headings come after the main heading.
#3306 would fix that. See also #3299.

* [ ] There are usually two `h1`s on the page, the wiki title with logo and the first heading in the main content. The preview view has even three `h1`s (the third being "Preview"). It is best practice to give the one heading the lowest level which describes the page, so usually the first heading in the main content. A lot of screen reader users navigate by heading, some would skip straight to the first `h1` in order to skip to the main content. That wouldn't work in this case. But the main thing is to stay consistent.

**Comment by SMSadegh19** [comment]
Hi! @selfthinker 
For my university course, I want to solve some issue about **Front-end**.

I think it can be a good-first-issue for me?
What do you think? can you help me to solve it?
